tests/DoctrineMongoODMModuleTest/Doctrine/CliTest.php

```diff
@@ -26,7 +34,7 @@
  * @link    http://www.doctrine-project.org/
  * @author  Adam Homsi <adam.homsi@gmail.com>
  */
-class CliTest extends PHPUnit_Framework_TestCase
+final class CliTest extends \PHPUnit_Framework_TestCase
```

Comment:
@alcaeus Are you saying that it is feasible and acceptable for someone to manually autoload test files and extend them, and therefore we must consider marking such test files as `final` a BC-break, but the only reason we don't is because the version number is less than one?